azure-pipelines.yml

```diff
@@ -44,47 +44,52 @@ steps:
     Get-ChildItem Env: | % { Write-Host "$($_.Key): $($_.Value)"  }
   displayName: Diagnostic Host Information
   name: hostinfo
+
+- powershell: |
+    . ./docker/ci/build.ps1
+    Lint-Dockerfile -Path ./docker/puppetserver-standalone/Dockerfile
+  displayName: Lint PuppetServer-Standalone Dockerfile
+  name: lint_puppetserverstandalone_dockerfile
 - powershell: |
     . ./docker/ci/build.ps1
-    Invoke-ContainerBuildSetup
-  displayName: Prepare Build Environment
-  name: build_prepare
+    Lint-Dockerfile -Path ./docker/puppetserver/Dockerfile
+  displayName: Lint PuppetServer Dockerfile
+  name: lint_puppetserver_dockerfile
 - powershell: |
     . ./docker/ci/build.ps1
-    Build-Container -Name puppetserver-standalone -Repository $ENV:BUILD_REPOSITORY
-  displayName: Build Puppetserver Standalone
-  name: build_puppetserver_standalone
+    Build-Container -Name puppetserver-standalone -Namespace $ENV:NAMESPACE
```

Comment:
[nitpick] you may be able to get rid of all the `-Namespace $ENV:NAMESPACE` since it looks like that's the default for the methods in the ps1.

Reply:
Yeah we're specifying namespace twice between build.ps1 and azure.yml in a few projects.

Reply:
Yeah, being explicit isn't a bad idea either FWIW